lib/garage_client/client.rb

```diff
@@ -85,9 +85,6 @@ def require_necessaries(options)
       if !options[:endpoint] && !default_options.endpoint
         raise "Missing endpoint configuration"
       end
-      if !options[:name] && !default_options.name
```

Comment:
Because `options[:name]` and `default_options.name` isn't used in Client class.
`Configuration#name` is used only in Configuration class itself, and the presence of `name` is validated there.